hatchery/pods.go

```diff
@@ -105,6 +131,19 @@ func statusK8sPod(userName string) (*WorkspaceStatus, error) {
 		return &status, nil
 	} else {
 		status.Status = "Launching"
+		conditions := make([]PodConditions, len(pod.Status.Conditions))
```

Comment:
It looks like this block is the same as the block above. Can this be broken out into a shared function?